src/static/js/earn.js

```diff
@@ -1130,50 +1207,88 @@ async function main() {
+        var poolShare = `<div class="col-sm-12 col-md-2 align-items-center text-center snob-tvl pb-10 pb-md-0">
         <p class="m-0 font-size-12"><ion-icon name="pie-chart-outline"></ion-icon> Your pool share is</p>
         <p class="m-0 font-size-16 font-weight-regular">${options.pool_share_display} </p>
-        <p class="m-0 font-size-12">(${options.user_pool_percent.toFixed(6)}%)</p>
-        </div>`;
+        <p class="m-0 font-size-12">${options.pool_share_display_pgl} </p>
+    </div>`;
       }
-      if (options.stake_display) {
+      var stakeDisplay = '';
+      if (options.stake_display ) {
+        //_print(options.stake_display)
         stakeDisplay = options.stake_display;
+      }else{
+        stakeDisplay = '';
       }
 
       var estimatedRate = `<div class="col-sm-12 col-md-2 align-items-center text-center snob-tvl pb-10 pb-md-0 mx-auto">
         <p class="m-0 font-size-12"> Estimated Rate</p>
-        <span class="badge badge-success font-size-12 px-5 px-sm-10 mx-10">${(2666 * options.user_pool_percent / 100 ).toFixed(2)} SNOB <ion-icon name="trending-up-outline"></ion-icon></span>
-        <p class="m-0 font-size-12">per day ($${(2666 * options.user_pool_percent / 100 * snobPrice).toFixed(2)})</p>
+        <span class="badge badge-success font-size-12 px-5 px-sm-10 mx-10">${(snowballsPerBlock * options.pool_weight * options.user_pool_percent / 100 / 1e18 * blocks24hrs).toFixed(2)} SNOB <ion-icon name="trending-up-outline"></ion-icon></span>
+        <p class="m-0 font-size-12">(24hr block rate)</p>
         </div>`;
 
       var earning = `<div class="col-sm-12 col-md-2 align-items-center text-center snob-tvl pb-10 pb-md-0">
         <p class="m-0 font-size-12"><ion-icon name="pie-chart-outline"></ion-icon> You are earning</p>
-        <p class="m-0 font-size-16 font-weight-regular">${(2666 * options.user_pool_percent / 100 ).toFixed(2)} SNOB </p>
-        <p class="m-0 font-size-12">per day ($${(2666 * options.user_pool_percent / 100 * snobPrice).toFixed(2)})</p>
+        <p class="m-0 font-size-16 font-weight-regular">${(snowballsPerBlock * options.pool_weight * options.user_pool_percent / 100 / 1e18 * 15000).toFixed(2)} SNOB </p>
+        <p class="m-0 font-size-12">per day ($${(snowballsPerBlock * options.pool_weight * options.user_pool_percent / 100 / 1e18 * 15000 * snobPrice).toFixed(2)})</p>
+        <p class="m-0 font-size-12">(Average block rate)</p>
         </div>`
     }
-    var availableStake = '';
+    availableStake = '';
     if ( options.display_amount > 0 ) {
-      availableStake = stakeUnstake(options.display_amount.toFixed(6), true, 'S3F');
+      availableStake = stakeUnstake(options.display_amount.toFixed(6), true, 'PGL');
     }
-    var availableUnstake = ''
-    if ( options.staked_pool / 1e18 > 0 ) {
-      availableUnstake = stakeUnstake((options.staked_pool / 1e18).toFixed(6), false, 'S3F');
+    availableUnstake = '';
+    if ( options.staked_pool.amount / 1e18 > 0 ) {      
```

Comment:
Can we replace this `options.staked_pool.amount / 1e18 > 0` by `options.staked_pool.amount > 0`

Reply:
This condition is original.